app/api/routers/token_bond.py

```diff
@@ -132,15 +132,18 @@ def list_all_straight_bond_tokens(
         stmt = stmt.where(IDXBondToken.is_redeemed == is_redeemed)
     count = session.scalar(select(func.count()).select_from(stmt.subquery()))
 
-    sort_attr = getattr(IDXBondToken, sort_item, None)
+    if sort_item == "created":
+        sort_attr = getattr(Listing, sort_item, None)
+    else:
+        sort_attr = getattr(IDXBondToken, sort_item, None)
```

Comment:
If API receive created sort_order query, records will be sorted by `Listing.created`, not sorted by `IDXBondToken.created`.